packages/components/src/DataTable/getNextFocus.ts

```diff
@@ -0,0 +1,78 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+export const getTabStops = (ref: HTMLElement): HTMLElement[] =>
+  Array.from(ref.querySelectorAll('td[tabindex="-1"],a,button'))
+
```

Comment:
Some helper functions known as [type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) to make TS happy:
```suggestion
const isTableCell = (
  element: Element
): element is HTMLTableCellElement | HTMLTableHeaderCellElement => {
  return ['TD', 'TH'].includes(element.tagName)
}
const isTableRow = (
  element: Element | null
): element is HTMLTableRowElement => {
  return element ? element.tagName === 'TR' : false
}
```